src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -249,5 +251,34 @@ internal static MauiAppBuilder RemapForControls(this MauiAppBuilder builder)
 
 			return builder;
 		}
+
+		internal static void AddDefaultMappings()
+		{
+#if GTK
+			ContentViewHandler.Mapper.AppendToMapping(
```

Comment:
I think will be better to manage and maintain to move this to FrameHandler.Gtk

Reply:
yes, this is already done in https://github.com/lytico/maui/pull/289 and part of [lytico/merge/main240321](https://github.com/lytico/maui/tree/lytico/merge/main240321)